example_data/epilepsy_bundle.json

```diff
@@ -0,0 +1,656 @@
+                  "display": "Uncertain significance"
+                }
+              ]
+            }
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "53034-0",
+                  "display": "inheritance"
+                }
+              ]
+            },
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "LA6705-0",
+                  "display": "AR"
+                }
+              ]
+            }
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://hl7.org/geneNames",
+                  "code": "gene-names",
+                  "display": "Gene Name"
+                }
+              ]
+            },
+            "valueString": " BRAT1"
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://hl7.org/evidence",
+                  "code": "evidence",
+                  "display": "Evidence for classification of variant"
+                }
+              ]
+            },
+            "valueString": "predicted null variant where loss of function is known disease mechanism (PVS1_very strong)."
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://hl7.org/mutation",
+                  "code": "confirmed-mutation",
+                  "display": "confirmed-mutation"
+                }
+              ]
+            },
+            "valueBoolean": false
+          }
+        ],
+        "note": [
+          {
+            "authorString": "comments",
+            "text": "This variant occurs in a recessive gene that has been 100% sequenced and no second variant identified."
+          }
+        ]
+      },
+      "request": {
+        "method": "POST",
+        "url": "Observation"
+      }
+    },
+    {
+      "fullUrl": "Patient/43a92b3e-d757-11ec-9d64-0242ac120002",
+      "resource": {
+        "id": "43a92b3e-d757-11ec-9d64-0242ac120002",
+        "resourceType": "Patient",
+        "active": true,
+        "name": [
+          {
+            "family": "Island",
+            "given": [
+              "London"
+            ]
+          }
+        ],
+        "gender": "male",
+        "birthDate": "2016-03-11",
+        "identifier": [
+          {
+            "value": "1146253",
+            "use": "official"
+          }
+        ]
+      },
+      "request": {
+        "method": "POST",
+        "url": "Patient"
+      }
+    },
+    {
+      "fullUrl": "Specimen/43a96978-d757-11ec-9d64-0242ac120002",
+      "resource": {
+        "resourceType": "Specimen",
+        "id": "43a96978-d757-11ec-9d64-0242ac120002",
+        "receivedTime": "2020-05-22T14:18:00",
```

Comment:
Have you gone through the PDF report yet? The received time is in the report and for you to structure it properly in FHIR using Specimen resource, you have to use an id and resourceType as is in the Bundle, so you need it.

Reply:
Ah is it, okay,  I thought we only had reporting and authorising dates - would you be able to send my some reports to my nhs email so that I can see how it's structured in the pdf?